### PR TITLE
[Issue #807] Move structured streaming to tutorials

### LINKED ISSUE #807: The default code won't compile in IDEA because of structure streaming
The default version in BigDL is Spark 1.5. There is some spark structure streaming code. They won't compile until changing the profile to spark 2.0/2.1.

### PR DESCRIPTION
**Pull request by yangw1234**
## What changes were proposed in this pull request?

move structured streaming to tutorials

## How was this patch tested?

existing tests and manually tested

## Related links or issues (optional)
fixed #807

### CODE REVIEW

**Comment by yangw1234** [comment]
pr validation pass

File: spark/dl/pom.xml [hunk 1]
```diff
@@ -252,17 +251,6 @@
                     </execution>
                 </executions>
             </plugin>
-
-            <plugin>
-                <groupId>net.alchim31.maven</groupId>
-                <artifactId>scala-maven-plugin</artifactId>
```
Comment (qiuxin2012) [inline spark/dl/pom.xml]: Why delete scala-maven-plugin?

The scala-maven-plugin is used for compiling/testing/running/documenting scala code in maven.

Comment (yangw1234) [inline spark/dl/pom.xml]: The scala-maven-plugin here is introduced to exclude the "structured streaming" files (see pr #876 ). There is another scala-maven-plugin configuration in "bigdl-parent.pom"

Comment (qiuxin2012) [inline spark/dl/pom.xml]: OK

**Comment by yiheng** [comment]
LGTM. Once jenkins test pass, can we merge it?

**Comment by yangw1234** [comment]
pr validation pass

**Pull request merged**